content/ui-patterns/saving.mdx

```diff
@@ -91,27 +98,40 @@ When defining these calls to actions, make sure to:
 
 For more guidance on button usage, see the [button interface guidelines](/ui-patterns/button-usage).
 
-<Box mb={3} display="flex" alignItems="flex-start" justifyContent="space-between" flexDirection={["column", "column", "column", "column", "row"]} sx={{gap: 3}}>
-    <Box mb={3} display="flex" alignItems="center" flexDirection="column" mx="auto" sx={{gap: 2}}>
-        <img
-            width="456"
-            src="https://user-images.githubusercontent.com/2313998/152432687-e119f44c-9325-4ae7-b277-e8559a4eb37b.png"
-        />
-        <Text as="p" mt="0" align="center" fontSize={1}>Controls all belong to one form</Text>
-        <img
-            width="456"
-            alt="A small form in a dialog that has 'Cancel' and 'Save' buttons at the bottom right"
-            src="https://user-images.githubusercontent.com/2313998/151623724-d59d5967-5484-4e81-a4c2-1109409956ee.png"
-        />
-        <Text as="p" mt="0" align="center" fontSize={1}>A form with a button to save and a button to cancel</Text>
-    </Box>
-    <Box mb={3} display="flex" alignItems="center" flexDirection="column" mx="auto" sx={{gap: 2}}>
-        <img
-            width="456"
-            src="https://user-images.githubusercontent.com/2313998/152432688-e74f67b2-a777-4c5f-be45-37a43615e6ab.png"
-        />
-        <Text as="p" mt="0" align="center" fontSize={1}>Controls divided into multiple sections</Text>
-    </Box>
+<Box
+  mb={3}
+  display="flex"
+  alignItems="flex-start"
+  justifyContent="space-between"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  sx={{gap: 3}}
+>
+  <Box mb={3} display="flex" alignItems="center" flexDirection="column" mx="auto" sx={{gap: 2}}>
+    <img
+      width="456"
+      src="https://user-images.githubusercontent.com/2313998/152432687-e119f44c-9325-4ae7-b277-e8559a4eb37b.png"
```

Comment:
Missing alt tag of presentation role, screen reader will attempt to read the URL.